google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketPanel.java

```diff
@@ -107,7 +107,7 @@ public void mousePressed(MouseEvent event) {
           }
         });
 
-    refresh();
```

Comment:
what if there is no active project? will refresh still get called by the project selector listener?

Reply:
Good catch, it will not set 'Unselected project' message in this case. Fixing.